sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -103,6 +103,7 @@ private[this] object SharedFactory {
   val jsonFactory = new JsonFactoryBuilder()
     // Enabled for Hive compatibility
```

Comment:
Can you update this comment, too, like `The two options below enabled for Hive compatibility`?

Reply:
l have updated this comment,thank you!